dracut/30ignition/mnt-boot.mount

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Ignition Boot Mount
+ConditionPathExists=/dev/disk/by-label/boot
+
+# Wait for multipathd to start to prevent racing over
+# access to the raw device. This ensures that device symlinks
+# are setup properly.
+Wants=multipathd.service
+After=multipathd.service
+
+# Make sure that the device exists
+Requires=dev-disk-by\x2dlabel-boot.device
+After=dev-disk-by\x2dlabel-boot.device
+
+# Shut down before switch-root
+Conflicts=initrd-switch-root.service
+
+[Mount]
+What=/dev/disk/by-label/boot
+Where=/mnt/boot
+Options=ro
```

Comment:
I guess for the finalization stage we have that do the `MountFlags=slave` + "remount rw" bit?

Reply:
AFAIK we don't do any remounting of /boot in the initramfs. I triple checked. It would only be needed if we moved in `systemd/ignition-firstboot-complete.service` which is after the initramfs.

Reply:
Yep, agreed.